bower.json

```diff
@@ -7,12 +7,13 @@
   "copyright": "hmgibson23",
   "main": "scribe-plugin-drag-and-drop.js",
   "dependencies": {
-    "scribe": "~1.3.2",
-    "scribe-common": "~0.0.11",
-    "lodash-amd": "~2.4.1",
     "requirejs": "~2.1.18"
   },
-  "devDependencies": {},
+  "devDependencies": {
```

Comment:
If I understand this PR these are no longer needed right? Can we just remove them rather than moving them to devDependencies (unless Im totally missing something)?